Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
+      activesupport (= 6.0.3.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activesupport (6.0.3.2)
+    activemodel (6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activerecord (6.0.3.3)
+      activemodel (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activesupport (6.0.3.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.6 → 1.1.7

[_[change-log](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/ruby-concurrency/concurrent-ruby)_]





<details>
<summary>Commits</summary>

A change of **42** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.1.6...v1.1.7).

These are the first 10 commits:
- (5652d30) [Remove unused line](https://github.com/ruby-concurrency/concurrent-ruby/commit/5652d30bf584268cc7a37c085f108796bc71e154)
- (50917ff) [Merge pull request #852 from fzakaria/remove-unused-line](https://github.com/ruby-concurrency/concurrent-ruby/commit/50917ff64fcc6e8b40bc53c870392d70abada705)
- (466f065) [Fix argument prefix warning](https://github.com/ruby-concurrency/concurrent-ruby/commit/466f0653f9d842f1c93319dc95a1565330e329c5)
- (2b5fd71) [Merge pull request #854 from bjfish/fix-argument-prefix-warn…](https://github.com/ruby-concurrency/concurrent-ruby/commit/2b5fd71683789da58955156bb496722de67fd899)
- (3bdbefb) [CI: Use JRuby 9.2.10.0](https://github.com/ruby-concurrency/concurrent-ruby/commit/3bdbefb3e7be3afc2164923f50b5c02607b6c498)
- (478c64e) [Do not allow failures on JRuby 9.2.10.0 Latest on Java 11](https://github.com/ruby-concurrency/concurrent-ruby/commit/478c64ee1825e8e405ae9df1aa0db52c5c90081d)
- (85f72c8) [Merge pull request #855 from olleolleolle/patch-2](https://github.com/ruby-concurrency/concurrent-ruby/commit/85f72c81583e9d551c43a604526ec28d747ff0b6)
- (c92d11e) [Introduce ThreadPoolExecutor without a Queue](https://github.com/ruby-concurrency/concurrent-ruby/commit/c92d11e6974161520a0cbeafdc60bf6d15191961)
- (658ef56) [Simply make queue always false when @synchronous](https://github.com/ruby-concurrency/concurrent-ruby/commit/658ef56620e61b907d273e5233be5149591aa93c)
- (bf824c9) [remove whitespace](https://github.com/ruby-concurrency/concurrent-ruby/commit/bf824c9c9121783a157465216faafd7a2e20938a)


</details>